### PR TITLE
blocks drag'n'drop uploading of grid thumbnails

### PR DESCRIPTION
**Pull request by tsop14**
Checks if URIs match pattern of Grid thumbnail images' URIs, only uploads images that do not match. 

NOW :v: 
uploads1

BEFORE 
uploads2

### CODE REVIEW

File: kahuna/public/js/upload/dnd-uploader.js [hunk 1]
```diff
@@ -21,9 +21,12 @@ dndUploader.controller('DndUploaderCtrl',
                             apiPoll, witnessApi) {
 
     var ctrl = this;
+    const gridThumbnailPattern = /https:\/\/media-service([0-9-a-z]+)thumbbucket([0-9-a-z]+)/;
```
Comment (theefer) [inline kahuna/public/js/upload/dnd-uploader.js]: The only thing about this pattern is that it's coupled with CF configuration that lives outside this repo. To make this more generic the API would have to tell kahuna what the domain our S3 buckets live under. Sounds like quite a faff, so maybe this is OK, but could we have a TODO to highlight that this is a bit of a hack?

**Comment by tsop14** [comment]
@theefer comment added.

**Comment by theefer** [comment]
:+1:

**Comment by jamesgorrie** [comment]
I wonder if we could pick up on the fact before showing the dnd window and just do nothing?

**Pull request merged**